UnityProject/Assets/Scripts/HealthV2/Living/BodyPartMutations.cs

```diff
@@ -247,6 +248,7 @@ public void OnDestroy()
 
 	public void ChangeToSpecies(PlayerHealthData PlayerHealthData, GameObject BodyPart)
 	{
+		if (this.TryGetComponent<Brain>(out var Brain)) return; //Make it a little bit harder to remove from a round
```

Comment:
what is this solving? Also
```suggestion
		if (this.TryGetComponent<Brain>(out var brain)) return; //Make it a little bit harder to remove from a round
```

Reply:
Stops people's brains getting transformed into random things, by mutation viruses